src/aap_eda/tasks/orchestrator.py

```diff
@@ -121,43 +125,139 @@ def _run_request(
     return True
 
 
-def _make_user_request(
-    process_parent_type: ProcessParentType,
-    id: int,
-    request_type: ActivationRequest,
-) -> None:
-    """Enqueue a task to manage the activation with the given id."""
-    requests_queue.push(process_parent_type, id, request_type)
-    job_id = _manage_process_job_id(process_parent_type, id)
-    unique_enqueue("activation", job_id, _manage, process_parent_type, id)
+class QueueNotFoundError(Exception):
+    """Raised when a queue is not found."""
+
```

Comment:
@Alex-Izquierdo Should we capture the queue name it might be helpful in debugging.